src/MainView.vala

```diff
@@ -129,24 +129,19 @@ public class DateTime.MainView : Switchboard.SettingsPage {
         manual_time_box.append (date_picker);
         manual_time_box.set_parent (manual_time_radio);
 
-        var week_number_label = new Gtk.Label (_("Show week numbers:")) {
-            halign = Gtk.Align.END
-        };
-
         var week_number_switch = new Gtk.Switch () {
-            halign = Gtk.Align.START,
-            valign = Gtk.Align.CENTER
+            valign = CENTER
         };
 
-        var week_number_info = new Gtk.Label (_("e.g. in Calendar and the Date & Time Panel indicator")) {
-            wrap = true,
-            xalign = 0
+        var week_number_label = new Granite.HeaderLabel (_("Show Week Numbers")) {
```

Comment:
Please change corresponding search result to avoid duplicating strings in weblate :)

Reply:
Oops, thanks for the reminder :sweat_smile: